Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/Pass.h

```diff
@@ -494,6 +500,9 @@ namespace AZ
             // the next Pass System update. The function internally build and initializes the pass.
             void ManualPipelineBuildAndInitialize();
 
+            // Registers any bindings specified as pipeline bindings with the pipeline for global reference
+            void RegisterPipelineConnections();
```

Comment:
super nit: Pipeline is the entity, but Global is the intent, perhaps call it 'RegisterGlobalConnections'?